doc/recog.bib

```diff
@@ -9,4 +9,35 @@ @incollection {IssacRecog
       YEAR = {2006},
    MRCLASS = {68P05 (20D99 68W30)},
   MRNUMBER = {MR2289128},
+      NOTE = {https://doi.org/10.1145/1145768.1145811},
+}
+
+@article {BlackBoxLie,
+  AUTHOR = {Babai, László and Kantor, William M. and Pálfy,
+            Péter P. and Seress, Ákos}
+   TITLE = {Black-box recognition of finite simple groups of Lie type by
+            statistics of element orders},
+ JOURNAL = {J. Group Theory},
+    YEAR = {2002},
+  VOLUME = {5},
+  NUMBER = {4},
+   PAGES = {383--401},
+ JOURNAL = {J. Group Theory},
+FJOURNAL = {Journal of Group Theory},
+    YEAR = {2002},
+    ISSN = {1433-5883},
+ MRCLASS = {20D06 (20P05)},
+MRNUMBER = {1931364},
+     DOI = {10.1515/jgth.2002.010},
+    NOTE = {https://doi.org/10.1515/jgth.2002.010},
+}
+
+@incollection {AltseimerBorovik2001,
```

Comment:
In the `.gi` file, you are using `AB01` instead. Maybe those should be made to match?